### PR TITLE
QA fixes

### PR DESCRIPTION
**Pull request by geramirez**
Multiple issues from: 18F/foia-hub/issues/538
- [x] fix the emails for the FLRA Dallas and Denver regional offices
- [x] update the FERC's submission link
- [x] remove duplicate emails
- [x] fix the JLP office url 

Also found that the email dups originate from the csv update [here](https://github.com/18F/foia/compare/missing_data?expand=1#diff-08719b9d357973b49df3d5a0eca9b445R88)

### CODE REVIEW

**Comment by geramirez** [comment]
I'm going to push this someplace else.

File: contacts/layer_with_csv.py [hunk 1]
```diff
@@ -85,9 +85,11 @@ def add_contact_info(contacts, row):
     address = organize_address(row)
     if address and 'address' not in office_struct:
         office_struct['address'] = address
-    if row['Email Address']:
-        office_struct['emails'].append(
-            row['Email Address'].replace('mailto:', ''))
+    email = row.get('Email Address')
```
Comment (khandelwal) [inline contacts/layer_with_csv.py:88]: So, does this only add to the existing list? How do we handle 'removes'?

**Comment by khandelwal** [comment]
Oh, nevermind. I see that this is closed.

File: contacts/layer_with_csv.py (hunk 1 above)
Comment (geramirez) [inline contacts/layer_with_csv.py:88]: We handle removes by rebuilding the dataset; however, I could add a set function. 
The new PR is here 18F/foia/pull/105

**Pull request closed without merging**